exporters/otlp/include/opentelemetry/exporters/otlp/otlp_grpc_exporter_factory.h

```diff
@@ -30,6 +33,13 @@ class OPENTELEMETRY_EXPORT OtlpGrpcExporterFactory
    */
   static std::unique_ptr<opentelemetry::sdk::trace::SpanExporter> Create(
       const OtlpGrpcExporterOptions &options);
+
+  /**
+   * Create an OtlpGrpcExporter using the given options and gRPC client.
+   */
+  static std::unique_ptr<opentelemetry::sdk::trace::SpanExporter> Create(
+      const OtlpGrpcExporterOptions &options,
+      nostd::shared_ptr<OtlpGrpcClient> client);
```

Comment:
It could cause confuse as `nostd` poniter are passed in, but `std` pointer is returned.

Reply:
I think we can use `nostd` here to avoid ABI break. Is it necessary for exporters?